docs/content/concepts/partitions-schedules-sensors/backfills.mdx

```diff
@@ -33,15 +33,27 @@ width={1737}
 height={335}
 />
 
-#### Single-run backfills
+### Single-run backfills
+
+By default, if you launch a backfill that covers `N` partitions, Dagster will launch `N` separate runs, one for each partition. This works well when code is single-threaded, as it avoids overwhelming it with large amounts of data.
+
+However, if you're using a parallel-processing engine like Spark and Snowflake, you may not need Dagster to help with parallelism. In this case, splitting the backfill into multiple runs can add extra overhead.
 
-By default, if you launch a backfill that covers N partitions, Dagster will launch N separate runs - one for each partition. This works well when your code is single threaded, because it avoids overwhelming it with large amounts of data. However, if you're using a parallel-processing engine like Spark and Snowflake, you often don't need Dagster to help with parallelism, so splitting up the backfill into multiple runs just adds extra overhead.
+Dagster supports backfills that execute as a single run that covers a range of partitions, such as executing a backfill as a single Snowflake query. After the backfill completes, Dagster will track that all the partitions have been filled.
```

Comment:
```suggestion
Dagster supports backfills that execute as a single run that covers a range of partitions, such as executing a backfill as a single Snowflake query. After the run completes, Dagster will track that all the partitions have been filled.
```
 "backfill" or "run" are basically synonyms in this use case, but I like "run" in this case since it it reinforces that there's only one run